src/Connection.php

```diff
@@ -1805,4 +1805,12 @@ public function exec(string $sql): int
     {
         return $this->executeStatement($sql);
     }
+
+    /**
+     * @psalm-return non-empty-string
+     */
+    public function getTypeRegistryName(): string
+    {
+        return $this->params['type_registry_name'] ?? 'default';
```

Comment:
Would move this to `TypeRegistry::DEFAULT_REGISTRY_NAME`, If feature is acceptable